src/layer/AbstractLayer.ts

```diff
@@ -41,4 +41,38 @@ export class AbstractLayer {
   ): Promise<PaginatedTiles> {
     throw new Error('Not implemented yet');
   }
+
+  public findFlyoverIntervals(tiles: Tile[]): FlyoverInterval[] {
+    if (!this.dataset || !this.dataset.orbitTimeMinutes) {
+      throw new Error('Orbit time is needed for grouping tiles into flyovers.');
+    }
+
+    let orbitTimeMS = this.dataset.orbitTimeMinutes * 60 * 1000;
+    let flyoverIntervals: FlyoverInterval[] = [];
+
+    let j = 0;
+    for (let i = 0; i < tiles.length; i++) {
```

Comment:
We talked about making this shorter, but I didn't find a shorter way of doing this.

Reply:
We could "code golf" this even more, but I think it is readable enough now (`i === 0` check, no tiles returned). One easy thing you could do is avoid nested `if`s:
```
      if (i === 0) {
        flyoverIntervals[j] = {
          fromTime: tiles[i].sensingTime,
          toTime: tiles[i].sensingTime,
        };
        continue;
      };
      // no need for "else"
```

But it's not important, feel free to resolve.

One concern though: are we certain that the tiles we get are sorted by `sensingTime`? Would it make sense to sort them explicitly?

Reply:
Thanks for both things.

>One concern though: are we certain that the tiles we get are sorted by sensingTime? Would it make sense to sort them explicitly?

Actually we don't know. We must not guess if the user of the library sent the sorted tiles to the function. So yes, it makes sense to sort them explicitly.

I added sorting of tiles and used you suggestion to avoid nested `if`s.

Reply:
Resolving.